src/components/ActionsBuilder/SupportedActions/common/Summary.tsx

```diff
@@ -25,14 +25,21 @@ const Summary = ({ decodedCall }: SummaryProps) => {
     return getNetworkById(chain?.id).nativeAsset.symbol;
   }, [chain]);
 
+  const isNativeTransfer = useMemo(
+    () => decodedCall?.callType === SupportedAction.NATIVE_TRANSFER,
+    [decodedCall?.callType]
+  );
+
+  const isRawTransaction = useMemo(
+    () => decodedCall?.callType === SupportedAction.RAW_TRANSACTION,
+    [decodedCall?.callType]
+  );
+

```

Comment:
Same here. I think you are not doing any hard logic or preventing any re-renders below.